### ISSUE TITLE
Request an example of app specific package.json setup

### ISSUE THREAD

**Issue by zergeborg** [nrwl/nx-examples#167]
I've seen that [a similar question was closed](https://github.com/nrwl/nx-examples/issues/61).

But the question remains: is it possible to add app/library specific package.jsons to an nx workspace?

Since this repo is [advertised on the nx website](https://nx.dev/latest/angular/examples/nx-examples) as the example for micro-frontends, I came here with high hopes, only to find disappointment since IMO this example is not a micro-frontend at all.

As I noted in #61 some thought leaders argue that it is fundamental for micro-frontends to be deployed autonomously, and a single version policy seriously hurts this goal.

**Comment by mattmbrightside** [comment]
I agree it is very confusing that there is a micro frontend page on the Nx site but they didn't make any effort to actually communicate how Nx could be used with a micro frontend.  I also spent 30 minutes working with this repo only to find out that it is not an example of a micro frontend as the site seems to imply. :(

**Comment by mattmbrightside** [comment]
The website appears to have been refactored over the last 1.5 years but the recipes section has some helpful info now: https://nx.dev/recipes/module-federation
